graceful/README.md

```diff
@@ -1,4 +1,4 @@
-## Graceful
+## Package `graceful` v1.0.1
```

Comment:
The latest version of `graceful` package is v1.0.0 :)

![image](https://user-images.githubusercontent.com/24525868/105052308-057f5500-5a70-11eb-807d-f24b7df83dd3.png)